.base_api

```diff
@@ -0,0 +1 @@
+e817932136f4f3894781bf32c5edf9b9863500d3
```

Comment:
What would be the suggested flow when we do introduce the breaking changes?
For example, now that we've release rc.1, we can still do breaking changes in rc.2 so when should we update `.base_api`? If we do it in the same PR that does the breaking change - we won't see CI warning us and listing the breakages. But if we don't - we'll have to update `.base_api` in subsequent PR which is uncomfortable.

Reply:
Just update it in the same pr if breakage is approved

Reply:
I would suggest then doing it in a separate commit, after CI does the run without such changes. It will allow to check if all the found breakages are intended and attach the report to the PR.

Reply:
That's reasonable. Good suggestion